docs/_static/images/overview.svg

```diff
@@ -0,0 +1,3 @@
le;" pointer-events="all" width="55" height="22" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; font-size: 10px; font-family: Helvetica; color: rgb(0, 0, 0); line-height: 1.2; vertical-align: top; overflow: hidden; max-height: 26px; max-width: 55px; width: 55px; white-space: normal; text-align: center;"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit;">Cloudwatch Logs</div></div></foreignObject><text x="28" y="16" fill="#000000" text-anchor="middle" font-size="10px" font-family="Helvetica">[Not supported by viewer]</text></switch></g><path d="M 93 262 L 231.84 340.93" fill="none" stroke="#000000" stroke-opacity="0.7" stroke-width="2" stroke-miterlimit="10" stroke-dasharray="6 6" pointer-events="none"/><path d="M 237.06 343.89 L 228.12 343.42 L 231.84 340.93 L 232.08 336.46 Z" fill-opacity="0.7" fill="#000000" stroke="#000000" stroke-opacity="0.7" stroke-width="2" stroke-miterlimit="10" pointer-events="none"/><g transform="translate(161,286)"><switch><foreignObject style="overflow:visible;" pointer-events="all" width="18" height="10" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; font-size: 10px; font-family: Helvetica; color: rgb(0, 0, 0); line-height: 1.2; vertical-align: top; overflow: hidden; max-height: 18px; max-width: 23px; width: 20px; white-space: normal; text-align: center;"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit;">logs</div></div></foreignObject><text x="9" y="10" fill="#000000" text-anchor="middle" font-size="10px" font-family="Helvetica">[Not supported by viewer]</text></switch></g><path d="M 437 257 L 288.15 341.92" fill="none" stroke="#000000" stroke-opacity="0.7" stroke-width="2" stroke-miterlimit="10" stroke-dasharray="6 6" pointer-events="none"/><path d="M 282.94 344.89 L 287.91 337.45 L 288.15 341.92 L 291.87 344.4 Z" fill-opacity="0.7" fill="#000000" stroke="#000000" stroke-opacity="0.7" stroke-width="2" stroke-miterlimit="10" pointer-events="none"/><g transform="translate(360,304)"><switch><foreignObject style="overflow:visible;" pointer-events="all" width="18" height="10" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; font-size: 10px; font-family: Helvetica; color: rgb(0, 0, 0); line-height: 1.2; vertical-align: top; overflow: hidden; max-height: 18px; max-width: 23px; width: 20px; white-space: normal; text-align: center;"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit;">logs</div></div></foreignObject><text x="9" y="10" fill="#000000" text-anchor="middle" font-size="10px" font-family="Helvetica">[Not supported by viewer]</text></switch></g></g></svg>
```

Comment:
Nice diagram!

It may be not worth it to do these things. However, if it is easy, I recommend
- adjusting the diagram so that the bulk of the flow is from left to right
- rearranging the :arrow_right: and :arrow_left: so that the arrow that happens first is on the top, and the arrow that happens after is on the bottom.